src/groundStation/groundStation.py

```diff
@@ -400,7 +404,7 @@ def getOptions(self):
 
         self.parser.add_argument(
             '-f',
```

Comment:
-f is used by the updater service which derives this class. This should have a different single character option, or just none at all

Reply:
@yunqingyi